boards/xtensa/esp32/doc/index.rst

```diff
@@ -91,6 +93,46 @@ message in the monitor:
 
    ***** Booting Zephyr OS vx.x.x-xxx-gxxxxxxxxxxxx *****
    Hello World! esp32
+   
+Building Blinky
+--------------------------
+
+Blinky is a sample application that simply makes the built-in led on your board 
+blink (see :ref:`blinky-sample`). It is a great starting point but it requires two
+tweaks in order to work on your ESP32 board:
+
+First, your prj.conf file should have the following Kconfig options:
+
+.. code-block:: none
+
+    CONFIG_GPIO=y
+    CONFIG_GPIO_ESP32=y
+
+Then, create a new file called esp32.overlay inside blinky's folder (samples/basic/blinky) 
+with this content:
+
+.. code-block:: devicetree
+
+    / {
+	    aliases {
+		    led0 = &led0;
+	    };
+
+	    leds {
+		    compatible = "gpio-leds";
+		    led0: led_0 {
+			    gpios = <&gpio0 2 GPIO_ACTIVE_HIGH>;
+			    label = "LED 0";
+		    };
+	    };
+    };
+
+Your board has one of its pins also connected to the built-in LED. 
```

Comment:
I would only change this line because esp32 default devkit has no built-in led.  
`If you board has a built-in LED or you wired LED, .....`.